src/fs.h

```diff
@@ -140,6 +140,21 @@ static inline path PathFromString(const std::string& string)
     return std::filesystem::path(string);
 #endif
 }
+
+/**
+ * Create directory (and if necessary its parents), unless the leaf directory
+ * already exists or is a symlink to an existing directory.
+ * This is a temporary workaround for an issue in libstdc++ that has been fixed
+ * upstream [PR101510].
+ */
+static inline bool create_directories(const std::filesystem::path& p)
```

Comment:
There is also another function:

```cpp
bool create_directories(const std::filesystem::path& p, std::error_code& ec);
```

Is it worth hijacking that too? In case somebody in the future decides to use that one instead?

Reply:
I thought about it, but defining a function that's not used felt wrong, somehow. Is there a way to un-define it?

Reply:
Cool, I did not know `= delete` could be used on non-member functions. It seems it can - https://en.cppreference.com/w/cpp/language/function#Deleted_functions. `static inline` can be skipped.

Reply:
Thanks, will remove the `static inline`.

Reply:
> `static inline` can be skipped.

> Thanks, will remove the `static inline`.

:smile: